tests/pytest/addons.py

```diff
@@ -58,6 +58,9 @@ def is_nvidia_gpu_present():
 using_memory_profiler = pytest.mark.skipif(_plugin_import('memory_profiler') is False,
                                 reason='Not detecting module memory_profiler. Install package if necessary and add to envvar PYTHONPATH')
 
+using_psi4 = pytest.mark.skipif(False,
+                                reason='Note detecting module psi4. Install package if necessary and add to envvar PYTHONPATH')
```

Comment:
```suggestion
                                reason='Not detecting module psi4. Install package if necessary and add to envvar PYTHONPATH')
```

Reply:
Good catch. And since you're probably wondering why `@using_psi4` is defined for a test in the psi4 test suite, the answer is so that I can use the same test file in another repo, where psi4 isn't mandatory.

Reply:
I suspected that may be the case, given your propensity to write reusable components.  I've never tried the suggested change thingy, but apparently it will let you click a button to accept the diff.  More cool stuff from GH :)

Reply:
ooh, I've never hit that button before. very nifty.